tasks/task1438_doqa_cooking_answer_generation.json

```diff
@@ -0,0 +1,1757 @@
+{
+	"Contributors": [
+		"Kuntal Kumar Pal"
+	],
+	"Source": [
+		"DoQA: (dataset: http://www.ixa.eus/node/12931, paper: https://arxiv.org/pdf/2005.01328.pdf)"
+	],
+	"Input_language": [
+		"English"
+	],
+	"Output_language": [
+		"English"
+	],
+	"Instruction_language": [
+		"English"
+	],
+	"Categories": [
+		"Question Generation"
+	],
+	"Definition": "Given a paragraph about cooking, and a set of conversational question answers about the paragraph, answer a follow-up question from the paragraph. The paragraph has a prefix 'CONTEXT:'. Each conversation question has a prefix Q: followed by answer prefix A: ",
+	"Positive Examples": [
+		{
+			"input": "CONTEXT: Oil definitely changes the crust, and whether or not it's 'better' depends on what you're going for. Oil in the dough tends to give a softer and thicker crust, while a lean dough tends to give a crisper and thinner crust. Other dough ingredients that tenderize include milk and eggs, and there are plenty of web pages listing 'dough enhancers' that give various effects. Again, it all depends on what type of crust and crumb you want.   Q: Does adding oil and fats affect bread crust development? A: Oil in the dough tends to give a softer and thicker crust  Q: What is the best oil or fat to add?",
+			"output": "Other dough ingredients that tenderize include milk and eggs",
+			"explanation": "Here the follow-up question asks about the alternative dough ingredients. The answer (milk and eggs) is mentioned in the passage."
+		},
+		{
+			"input": "CONTEXT: In Ireland boiled or glazed ham is a staple. Traditionally it's had with cut up cabbage (ideally you would cook the cabbage in the same water the ham was boiled in as this gives it lovely flavour) but you could just cut it up and pan fry it with a little butter. Normally it would be served with boiled floury potatoes. Traditionally (in Ireland) you wouldn't have a sauce with it but a parsely sauce (as stated in the other reply would be lovely) or simply serve with some strong mustard or a brown sharp sauce such as HP. You could also shred the ham and serve it with noodles in a hot broth with some shredded cabbage and other vegetables.The ham is going to be centre of attention - salty, rich with a sweetness. The cabbage and potato are happy bedfellows as the cabbage cuts through the saltyness/sweetness. You don't want strong flavours competing with the ham and neither cabbage or potato will.   Q: Honey-glazed gammon (ham) - What to serve with it? A: Normally it would be served with boiled floury potatoes  Q: I was going to go with a selection of steamed vegetables but would these also need a sauce/gravy?",
+			"output": "Traditionally (in Ireland) you wouldn't have a sauce with it but a parsely sauce",
+			"explanation": "The answer to the question is directly mentioned in the passage (parsely sauce)"
```

Comment:
Please add a period at the end of the sentence (same for the second negative example).